pkged.go

```diff
@@ -1,10 +1,8 @@
-// Code generated by pkger; DO NOT EDIT.
-
 package main
```

Comment:
Why is this file in the repo? Can it be generated by anyone?

Reply:
Yes, but the user would need to install pkger globally first and then generate it. This way, you just build and get the app running. The bundled is already on the master branch, currently it bundles just the fonts.

Reply:
We can't have a generated binary checked into the repo. Each dev should have pkger and generate the binary locally(instructions should be added to the readme). Adding binary to commits will make the repo heavy in size.